src/retasc/validator/generate_schema.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+import yaml
+
+from retasc.validator.models import Rule
+
+
+def generate_schema(output_path="src/retasc/validator/schemas/rules_schema.yaml"):
```

Comment:
The default value is too specific, better to leave it out:

```suggestion
def generate_schema(output_path: str) -> None:
```

Reply:
Done